### PR TITLE
endangered species component

### LINKED ISSUE #9: Endangered species component
Refer to the [Monitor Log panel](https://www.figma.com/file/Hd4IWsfe6KbDkB4f3iSskL/%5BDeveloper%5D-Wireframes?node-id=25%3A5366) in the Volunteer wireframes, and create the endangered species component.

Create both the [popup](https://www.figma.com/file/Hd4IWsfe6KbDkB4f3iSskL/%5BDeveloper%5D-Wireframes?node-id=25%3A5970) as well as the preview component on the Monitor Log panel. Examples of this are the Snowy Plowers and California Least Tern sections.

Reminder that we're using Chakra's UI kit! Here's their [documentation](https://chakra-ui.com/docs/getting-started).

### PR DESCRIPTION
**Pull request by ctc-devops**
Refer to the [Monitor Log panel](https://www.figma.com/file/Hd4IWsfe6KbDkB4f3iSskL/%5BDeveloper%5D-Wireframes?node-id=25%3A5366) in the Volunteer wireframes, and create the endangered species component.

Create both the [popup](https://www.figma.com/file/Hd4IWsfe6KbDkB4f3iSskL/%5BDeveloper%5D-Wireframes?node-id=25%3A5970) as well as the preview component on the Monitor Log panel. Examples of this are the Snowy Plowers and California Least Tern sections.

Reminder that we're using Chakra's UI kit! Here's their [documentation](https://chakra-ui.com/docs/getting-started).
closes #9

### CODE REVIEW

**Comment by ZubairSidhu** [comment]
Small note: Moving the footnote/dropdown text into their own files is a really good idea, but I would move `common/FootNotes.js` and `common/DropdownOptions.js` into `src/components/EndangeredSpecies/`, as the text is likely to only be used in that component. Great work otherwise!

File: src/components/EndangeredSpecies/FootNotes.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    'or more color per band) = yellow, blue on left and blue/green split and orange on ' +
+    'right = yb:(b/g)o. Color Code: Aqua:a, Blue:b, Green:g, Black:k, ' +
+    'Brown/Tan/Buckskin:n, Lime:l, Orange:o, Pink:p, Red:r, Silver:s, Violet:v, ' +
+    'White:w, Yellow:y. Only use x if one leg is unbanded and the other is banded. ' +
+    'Sex: Male, Female, Unknown',
+  nest: 'C/N = copulation/mating and nest construction, I = incubation, H = hatchling, B = brooding, O = other (specify in notes)',
+  behavior:
+    'R = roosting, F = foraging, S = sitting as if on nest, C=courting (scraping, ' +
+    'dancing, & fish presentation, shell tossing), Fishing = F, Flying = FL, Flushing = ' +
+    'FLU, Distraction Behavior = DB, Feeding = FE, O = other (specify in notes)',
```
Comment (PhillipChn) [inline src/components/EndangeredSpecies/FootNotes.js:19]: Based off the figma, the dropdown menu options are displayed in full, so should we keep the abbreviations for the tootip? (For sex, nesting, and behavior)

File: src/components/EndangeredSpecies/BandsSex.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  Input,
+  Tooltip,
+  Select,
+  Heading,
+  FormControl,
+  FormLabel,
+} from '@chakra-ui/react';
+import footNotes from './FootNotes';
+
+function BandsSex({ totalAdults, totalFledges }) {
```
Comment (ZubairSidhu) [inline src/components/EndangeredSpecies/BandsSex.jsx]: For consistency, please use arrow functions whenever possible. ex: `const BandsSex = ({ totalAdults, totalFledges }) => {`

**Review by janehvo — DISMISSED** [review]
LGTM! :)

**Review by ctian1 — APPROVED** [review]

**Pull request merged**